spring-security/src/main/java/com/epam/esm/user/dto/UserDTO.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.esm.user.dto;
+
+import com.epam.esm.orders.model.Order;
+import com.epam.esm.role.Role;
+import com.epam.esm.user.model.User;
+import lombok.*;
+import org.springframework.hateoas.RepresentationModel;
+import org.springframework.stereotype.Component;
+
+import java.util.Set;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class UserDTO extends RepresentationModel<UserDTO> {
+    private long id;
+    private String username;
```

Comment:
What's DTO in this context? Request model?

Reply:
Response model , due to , I shouldn't return model with password for example

Reply:
Then I would like to ask you to rename it to response instead of DTO. DTO is more like a concept type of a model, using it in a class name doesn't bring a lot of value IMO

Reply:
okey, i understood

Reply:
i remade this one